### PR TITLE
[SWT-41] Drop System.Drawing.Common library and .NET Framework 4.5.1

### PR DESCRIPTION
**Pull request by CrDoming**
[[SWT-41](https://pyramid.atlassian.net/browse/SWT-41)]

The `SkiaSharp` library has replaced the `System.Drawing` library so UWP apps can utilize `ThermalTalk`. Since `SkiaSharp` does not support .NET Framework 4.5.1, that framework was dropped. Additionally, the 3 sample apps were combined into 1 command-line interface.

[SWT-41]: https://pyramid.atlassian.net/browse/SWT-41?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codygillespie** [comment]
@CrDoming I am running into an issue with barcode printing via the CLI.

Command used: 
`printcode phoenix COM14 ABCDEF`

Print I am seeing: 
image

**Comment by codygillespie** [comment]
> > @CrDoming I am running into an issue with barcode printing via the CLI.
> > Command used: `printcode phoenix COM14 ABCDEF`
> > Print I am seeing: image
> 
> Is the firmware on your Phoenix printer version 1.9 or newer?

This was on Phoenix 1.08 UNV firmware

**Comment by CrDoming** [comment]
@codygillespie 

Printing a raster image is only supported on Phoenix firmware versions 1.9 and newer, according to https://escpos.readthedocs.io/en/latest/imaging.html.

File: ThermalTalk/ThermalTalk.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <Version>2.1.1.0</Version>
```
Comment (codygillespie) [inline ThermalTalk/ThermalTalk.csproj:5]: Let's move to v3

Comment (codygillespie) [reply ThermalTalk/ThermalTalk.csproj:5]: And lets add a section about v3 in the readme as well as a statement on what .NET framework users should do in order to use thermaltalk (i.e. use v2)

**Review by codygillespie — CHANGES_REQUESTED** [review]

**Review by codygillespie — APPROVED** [review]

**Pull request merged**